commerce_coordinator/apps/lms/tasks.py

```diff
@@ -54,6 +54,11 @@ def fulfill_order_placed_send_enroll_in_course_task(
                 'name': 'order_number',
                 'value': order_number,
             },
+            {
+                'namespace': 'order',
+                'name': 'line_item_id',
+                'value': line_item_id,
+            },
```

Comment:
@MushtaqRossier @grmartin One of the ACs of the ticket was to add CT's Order ID in enrollment attributes as well along with order number

`ENSURE Coordinator’s LMS client for fulfilling an order sends a new attribute called order_id as a sibling of order_number with value Order.id`

Can you please add that too?